integration_test/logged_test.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:integration_test/integration_test.dart';
+import 'package:wutsi_wallet/main.dart' as app;
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+
+Future<void> restoreFlutterError(Future<void> Function() call) async {
+  final originalOnError = FlutterError.onError!;
+  await call();
+  final overriddenOnError = FlutterError.onError!;
+
+  // restore FlutterError.onError
+  FlutterError.onError = (FlutterErrorDetails details) {
+    if (overriddenOnError != originalOnError) overriddenOnError(details);
+    originalOnError(details);
+  };
+}
+
+Future<void> main() async {
+  final binding = IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+  if (binding is LiveTestWidgetsFlutterBinding) {
+    binding.framePolicy = LiveTestWidgetsFlutterBindingFramePolicy.fullyLive;
+  }
+
+  group(('WU-105'), () {
+
+    testWidgets('normal user account update WU-105',
+            (WidgetTester tester) async {
+//setup
+            app.main();
+
+          await tester.pumpAndSettle();
+          await Future.delayed(const Duration(seconds: 5));
+
+          await tester.pumpAndSettle();
+          final Finder phonefield = find
+              .byType(TextField)
+              .first;
+          final Finder nextbutton = find.text('Next', findRichText: true);
+
+//do
+          await Future.delayed(const Duration(seconds: 1));
+          await tester.enterText(phonefield, "693842356");
```

Comment:
Use instead `670000010` as phone number.
This is one of the phone test phone number configured in the app.
The pasword is `111111`

Reply:
I use 670000010 for user account creation but we have 690904938 , 696074190, et 693842356 which can also pass through otp verification